test/Profiler/coverage_relative_path_exec.swift

```diff
@@ -0,0 +1,25 @@
+// REQUIRES: profile_runtime
+// REQUIRES: executable_test
+// REQUIRES: OS=macosx
+
+// This test is to make sure llvm-cov can deal with a coverage-prefix-map.
+
+// To make sure this test is resilient to directory changes, we create nested directories inside of the
+// temporary test directory and assert those exist, or don't exist, in the emitted ir
+//
+// RUN: %empty-directory(%t)
+// RUN: mkdir -p %t/root/nested
+// RUN: echo "func coverage() {}" > %t/root/nested/coverage_relative_path.swift
+// RUN: cd %t/root
+
+// RUN: %target-build-swift -profile-generate -profile-coverage-mapping -Xfrontend -coverage-prefix-map -Xfrontend %/t/root=. -Xfrontend -disable-incremental-llvm-codegen -o %t/main %/t/root/nested/coverage_relative_path.swift
```

Comment:
`-Xfrontend %/t/root=.` should be `-Xfrontend %t/root=.`?

Reply:
Good catch! I copy and pasted this from the other file, seems we should fix it there too